packages/http-client-csharp/emitter/src/lib/client-model-builder.ts

```diff
@@ -184,7 +184,6 @@ export function createModel(sdkContext: CSharpEmitterContext): CodeModel {
         Type: parameterType,
         Location: RequestLocation.Uri,
         IsApiVersion: parameter.isApiVersionParam,
-        IsResourceParameter: false,
```

Comment:
I removed this parameter because in everywhere of our emitter, we are assigning `false` to this property, and in all the places in MGC.
In autorest.csharp, we do not even have this property on `InputParameter` class.